webapp/channels/src/components/suggestion/suggestion.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React, {useCallback} from 'react';
+
+export interface SuggestionProps<Item> {
+    // eslint-disable-next-line react/no-unused-prop-types
+    item: Item;
```

Comment:
ESLint was complaining about this being an unused prop type even though this type is used for the props of other `*Suggestion` components